### ISSUE TITLE
Unable to install Xcode. Xcode.xip is damaged and can't be expanded.

### ISSUE THREAD

**Issue by fandiluo** [RobotsAndPencils/XcodesApp#214]
**Describe the bug**
The archive "Xcode-13.3.1+13E500a.xip" is damaged and can't be expanded.

**Screenshots**
<img width="900" alt="Screen Shot 2022-04-26 at 3 59 57 PM" src="https://user-images.githubusercontent.com/9468958/165251904-eb49cfd9-f949-4781-abfd-0f02a9715061.png">

**Version**
 - OS:  12.3.1

I reinstalled the operating system and xcodes, but I still get this error. How do I troubleshoot this problem?

**Comment by MattKiazyk** [comment]
Hi @fandiluo 
If you browse to `/users/%username%/Library/Application Support/com.robotsandpencils.XcodesApp` you'll see the xip file there. So a couple of questions on that

- Is it the regular size (something like 12 GB)?
- if you double click it, does it give you an error?

If it's a small size, can you delete it, make sure you are logged into your Apple account on Xcodes and try again? 
If it gives you an error while expanding, it might give you the better error. 

Thanks. Let me know

**Comment by fandiluo** [comment]
> Is it the regular size (something like 12 GB)?
Yes, I have confirmed it. (Xcode 13.3 ~= 10.09 GB)

> if you double click it, does it give you an error?
After I clicked on it, I got the same error


I'm sure I'm logged in...
I have compared the file sizes downloaded from the official site are the same.
I also tried using safari browser to download from the official site. I got the same error when I manually extracted the files.
I don't think it's a problem with XcodesApp, but I can't find a way to fix it.

Thanks for your help!  I hope we can find a solution 👍
